qsirecon/interfaces/dipy.py

```diff
@@ -589,3 +607,66 @@ def _run_interface(self, runtime):
             self._results[metric] = out_name
 
         return runtime
+
+
+class _KurtosisReconstructionMicrostructureInputSpec(DipyReconInputSpec):
+    kurtosis_clip_min = traits.Float(-0.42857142857142855, usedefault=True)
+    kurtosis_clip_max = traits.Float(10.0, usedefault=True)
+
+
+class _KurtosisReconstructionMicrostructureOutputSpec(DipyReconOutputSpec):
```

Comment:
There should also be a dkimicro KFA parameter

Reply:
I don't see that in the API docs (https://docs.dipy.org/1.0.0/reference/dipy.reconst.html#dipy.reconst.dki_micro.KurtosisMicrostructuralFit). Is it just not documented?

Reply:
Let me check.. Couple of thoughts:
1. Those are the 1.0.0 docs
2. It's not listed in the 1.9.0 docs (which is what we're running).. However, none of the kurtosis params are and yet you get them from that fit method.
3. It is listed in the 1.10 and 1.11 docs

Reply:
Oh awesome. I'll add KFA then.

Reply:
It should be added now